backend/config/curation-api.yml

```diff
@@ -818,10 +930,14 @@ components:
         description: A URL that links associated resources with the collection.
         properties:
           link_name:
+            description: A brief description of the URL.
             nullable: true
             type: string
           link_type:
-            description: If the link_type is DOI, then the `link_url` must be a valid DOI link and the `link_name` will be ignored or overwritten if provided. For example, https://doi.org/00.0000/science.abc1234 is a valid DOI `link_url`. All other links types can be any valid URL.
+            description: |
+              If the link_type is DOI, then the `link_url` must be a valid DOI link and the `link_name` will be ignored 
+              or overwritten if provided. For example, https://doi.org/00.0000/science.abc1234 is a valid DOI 
+              `link_url`. All other links types can be any valid URL.
```

Comment:
@brianraymor how do you want to handle the inherent conflict between the name of this field and the fact that we will only be accepting (and returning) DOI curie references, and not actual urls?

Should we just specify that we expect the user to provide a CURIE reference for the `"link_url"` field?

Reply:
I responded to this question with a proposal in the related [curation-api](https://czi-sci.slack.com/archives/C02UGUV5R3J/p1662512390544879) thread.